configBase.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const axios = require('axios')
+const FormData = require('form-data')
+const fs = require('fs')
+const path = require('path')
+
+const SERVICE_NAME = process.env.SERVICE_NAME
+const API_PASSWORD = process.env.API_PASSWORD
+const SERVICE_API_PORT = process.env.SERVICE_API_PORT
+const API_PATH = process.env.API_PATH
+const API_USERNAME = process.env.API_USERNAME
+const SSL = process.env.SSL
+const MIME_TYPE = process.env.MIME_TYPE
+
+let data
+switch (MIME_TYPE) {
+  case 'multipart/form-data':
+    data = new FormData()
+    data.append('form', fs.createReadStream(path.resolve(__dirname, 'export.jsrexport')))
+    break;
+
+  case 'JSON':
+    const jsonData = JSON.parse(
+      fs.readFileSync(path.resolve(__dirname, 'openhim-import.json'))
+    )
+    data = JSON.stringify(jsonData)
+    break;
+
+  default:
+    break;
+}
+
+const protocol = SSL == 'false' ? 'http' : 'https'
+const dataHeaders = data?.getHeaders() ?? {}
+
+let config = {
+  method: 'post',
+  url: `${protocol}://${SERVICE_NAME}:${SERVICE_API_PORT}${API_PATH}`,
+  auth: {
+    username: API_USERNAME,
+    password: API_PASSWORD
+  },
+  headers: {
+    'Content-Type': 'application/json',
```

Comment:
this can probably read from the MIME_TYPE variable

Reply:
Indeed it can... [1d6d4bf](https://github.com/jembi/api-config-importer/pull/1/commits/1d6d4bfcae3f173d4e644e8be970e32896265d29)